drivers/hmis/app/models/hmis/hud/custom_assessment.rb

```diff
@@ -65,7 +65,9 @@ class Hmis::Hud::CustomAssessment < Hmis::Hud::Base
   scope :post_exits, -> { where(data_collection_stage: 6) }
 
   scope :with_role, ->(role) do
-    stages = Array.wrap(role).map { |r| Hmis::Form::Definition::FORM_DATA_COLLECTION_STAGES[r.to_sym] }.compact
+    stages = Array.wrap(role).map do |r|
+      Hmis::Form::Definition::FORM_DATA_COLLECTION_STAGES.merge({ CUSTOM_ASSESSMENT: 99 })[r.to_sym]
```

Comment:
I was wondering whether we want to just incorporate `CUSTOM_ASSESSMENT` directly as a key into `FORM_DATA_COLLECTION_STAGES`. It seems that right now we have to accommodate for it in every usage of this map.
https://github.com/greenriver/hmis-warehouse/blob/185b4403bd2b23ce876ad7d61360979a49a81117/drivers/hmis/app/graphql/types/hmis_schema/assessment.rb#L56
https://github.com/greenriver/hmis-warehouse/blob/185b4403bd2b23ce876ad7d61360979a49a81117/drivers/hmis/app/models/hmis/hud/custom_assessment.rb#L68
https://github.com/greenriver/hmis-warehouse/blob/185b4403bd2b23ce876ad7d61360979a49a81117/drivers/hmis/app/models/hmis/hud/custom_assessment.rb#L178